internal/pkg/cli/svc_init.go

```diff
@@ -255,14 +263,18 @@ func (o *initSvcOpts) newLoadBalancedWebServiceManifest() (*manifest.LoadBalance
 }
 
 func (o *initSvcOpts) newBackendServiceManifest() (*manifest.BackendService, error) {
+	dfPath, err := o.getRelativePath()
+	if err != nil {
+		return nil, err
+	}
 	hc, err := o.parseHealthCheck()
```

Comment:
Just to double check this is working fine right?

Reply:
Yes. :)